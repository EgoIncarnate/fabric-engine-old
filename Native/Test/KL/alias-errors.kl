/*
 *  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
 */

alias UnregisteredType AliasTypeOne;
alias Integer AliasTypeTwo;
alias Scalar AliasTypeTwo;
