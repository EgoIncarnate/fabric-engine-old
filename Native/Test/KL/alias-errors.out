1:1: error: no registered type named 'UnregisteredType'
3:1: error: type 'AliasTypeTwo' already registered with a different definition
